### DISCUSSION TITLE
Google Analytics doesn't work with 'G-XXXXXXXXXX' tag id.

### DISCUSSION THREAD

**Discussion by rexdivakar** [techno-tim/techno-tim.github.io#153] [Questions] [answered]
Hey Tim, I tried replicating the templates but I'm not able to configure Google Analytics can you please share how you did it ?

**Comment by timothystewart6** [comment, marked as answer]
see this line
https://github.com/techno-tim/techno-tim.github.io/blob/master/_config.yml#L71

**Comment by rexdivakar** [comment]
weird I have already done it, still my dashboard isn't populating any values. Any suggestions ?

https://github.com/rexdivakar/rexdivakar.github.io/blob/master/_config.yml#L61

![image](https://user-images.githubusercontent.com/15235122/172505001-7a9697cc-a5b0-4679-a456-5dc2f4a51062.png)

**Comment by timothystewart6** [comment]
not sure, I would just check the network tab in your browser (f12) and see if you can see the requests going to google.  If you're running and ad block or pihole, it will block it

**Reply by rexdivakar** [reply]
I do have a Pihole running so I disabled it and cleared my local DNS and tried again to find the requests were getting blocked, Do you have any idea how to fix this ?

![image](https://user-images.githubusercontent.com/15235122/172523722-a599188e-e94f-466d-8103-df946e6dcb96.png)

**Comment by timothystewart6** [comment]
something in your browser or DNS is blocking it, again it's pihole or an adblocker, or something else.  Disable everything then work backwards
